### PR TITLE
refactor: fix issues with docs scripts

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: scripts/publish/docs.js [hunk 1]
```diff
@@ -107,19 +96,17 @@ function checkNameLinks(files) {
...
+  .then(() => console.log(`Wiki path: ${outputPath}`))
+  .then(() => console.log('Cloning...'))
+  .then(() => execute(`git clone "https://github.com/angular/angular-cli.wiki" ${outputPath}`))
+  .then(() => console.log('Copying Files...'))
+  .then(() => createFiles())
+  .then(() => process.chdir(outputPath))
+  .then(() => console.log('Committing...'))
+  .then(() => execute('git add .'))
+  .then(() => execute(`git commit -m "${version}"`))
+  .then(() => execute('git push'));
```
Comment (sumitarora) [inline scripts/publish/docs.js]: misplaced `;`

File: scripts/publish/docs.js [hunk 2]
```diff
@@ -2,25 +2,16 @@
 'use strict';
 
 /* eslint-disable no-console */
-const fs = require('fs-extra');
+const exec = require('child_process').exec;
+const fs = require('fs');
```
Comment (hansl) [inline scripts/publish/docs.js]: No, we don't use anything from `fs-extra` anymore.

Comment (hansl) [inline scripts/publish/docs.js]: Note that I changed `outputFile` to `writeFile`. Since there's no directories in the wiki they're identical.

Comment (sumitarora) [inline scripts/publish/docs.js]: 👍

**Pull request merged**